java/code/src/com/redhat/rhn/common/db/datasource/xml/Package_queries.xml

```diff
@@ -263,7 +263,7 @@ SELECT C.name,
        CS.checksum_type checksumtype,
        F.file_mode,
        F.linkto,
-       TO_CHAR(F.mtime, 'YYYY-MM-DD HH24:MI:SS') AS MTIME
```

Comment:
These seem to cause some problems with the time displayed (because we cut of the timezone information here) if server and user preferred timezone are different. So we should probably remove all of these instances. I already took care of 2 other of these in https://github.com/uyuni-project/uyuni/pull/4496. But we can create a separate issue to take care of these.

Reply:
:+1:  Yes, let's do it one after another.